### DISCUSSION TITLE
area 2-13 high rounds and records

### DISCUSSION THREAD

**Discussion by Ellis243454** [nzp-team/nzportable#663] [Records and Speedruns]
I didn't see one for this map so I thought I would make one my highest round so far in this map is 28 it isn't much but if I didn't leave my camping spot and was able to keep a steady stream of ammo for my ppsh I could have gotten way higher 

![2024-02-16_17-41-22.420_top.jpg](https://github.com/nzp-team/nzportable/assets/132875822/4c328b5c-0724-4d42-b7e7-77403ecbed70)

**Comment by DerpedCrusader** [comment]
Man, I tried to make the ppsh place where it was an ordeal to get ammo for it, and you couldn't get it easily. or did you just keep getting max ammos at good intervals

**Reply by Ellis243454** [reply]
I couldn't reach it most of the time that is actually what killed me if I wasn't looking in the right spot I wouldn't even get the prompt to buy it when falling down the elevator shaft I had to resort to using the Thompson and that didn't last long I used the hallway with jug as a camping spot and didnt buy the one door that lead to speed and staminup and used that as a small area to camp but missing the ppsh wallbuy when I needed ammo made the trip back to that camping spot harder than it was to just stay there

**Comment by pahntomforces-guy** [comment]
im gonna set a record

**Comment by DGxShAdOwZz** [comment]
![20240428131642_1](https://github.com/nzp-team/nzportable/assets/128944577/e21c5028-d8ce-4d3e-bbae-8d2566629faf)
Would have gotten into the 40s if i didnt lose focus, hopefully ill get higher the more i play
